src/parser/priest/discipline/modules/spells/Penance.tsx

```diff
@@ -89,4 +88,12 @@ class Penance extends Analyzer {
   }
 }
 
+export interface PenanceDamageEvent extends DamageEvent {
+  penanceBoltNumber: number;
+}
+
```

Comment:
(referring to this comment in other comments)

```typescript
export function IsPenanceDamageEvent(event: DamageEvent): event is PenanceDamageEvent {
  return (event as PenanceDamageEvent).penanceBoltNumber !== undefined;
}
```
 and the same for the heal event

Reply:
I can't typecheck this at function call time as the field is added 100% in this module. Are you saying that I should add IsPenanceDamageEvent typechecking for the other modules that use the penanceBoltNumber?

Reply:
Ignore that, I get it now :D